src/common/app.config.js

```diff
@@ -356,7 +356,13 @@ export const appConfig = /* @ngInject */ function (envServiceProvider, $compileP
     RADIO_STATION: 'Radio Station',
     RADIO_STATION_LIST_ERROR: 'There was an error loading radio stations in your area.',
     RADIO_STATION_SELECT_ERROR: 'There was an error selecting a radio station.',
-    PREFERRED_RADIO_STATION: 'Preferred Radio Station: '
+    PREFERRED_RADIO_STATION: 'Preferred Radio Station: ',
+    SPOUSE_DETAILS_TOOLTIP: 'By adding your spouse\'s name, we can ensure that both of your contributions are linked together, providing a clearer and more accurate record of your household\'s giving.',
+    SECURE: 'Secure',
+    ADD_SPOUSE: 'Add Spouse',
+    REMOVE_SPOUSE: 'Remove',
+    SUGGESTED_AMOUNT_HELP: 'Suggested Gift Amounts. Tab for Custom Amount',
+    SUGGESTED_AMOUNT: 'Suggested Amount'
```

Comment:
Did you mean this to match the label in the template?

```suggestion
    CUSTOM_AMOUNT: 'Custom Amount'
```

Reply:
Note, suggested amounts and custom amounts are not the same.

Reply:
I was thinking this would be for a custom suggested amount. Is that correct?